[WIP] Qt 5.12.9

MoJo2600 opened the pull request:
I'm trying to compile [welle.io](https://github.com/AlbrechtL/welle.io/blob/master/RASPBERRY-PI.md) for raspberry pi and stumbled upon this great project. But I want to compile it with 5.12 LTS. I adjusted your code to work, but right now I'm only focusing on Raspbery Pi 3, only full init and disregarded backward compatibility right now, hence WIP.

My current status is:
- Upgraded Crosscompiler to lenaro 7.5 (I have to upgrade the download script)
- Upgraded raspi image to buster
- Qt 5.12.9 builds successfully
- Qt modules build successfully

Still working on it

Commit:
upgrade scripts tp 5.12.9

Commit:
qt is building

Commit:
all modules compiled

MoJo2600 committed:
mpg123.h missing

MoJo2600 committed:
sucessfully compiled welle.io

MoJo2600 committed:
updated toolchain

MoJo2600 committed:
docker create

synapticvoid commented:
Hello Christian,

Wow thanks a lot for this awesome contribution! I've skimmed through your changes and it looks very promising. Ping me again when you have stabilized your changes and I'll gladly review and test your PR.

We might have some back and forth about a few fixes but I definitely want to merge your work :+1: !

MoJo2600 commented:
Yes, it is very crude right now, as I just figured out how everything is working. Have to do some cleanup later. I managed to build my intended application welle.io and it is running on my raspberry so the build is definitely working.

MoJo2600 committed:
cleanup

MoJo2600 committed:
fixed linaro path

MoJo2600 commented:
Okay, i did some cleanup and removed the requirements for welle.io. I tested it with rpi3 and 5.12.9. I will try to compile the old QT versions tomorrow. But I think we can start to talk about the fixes you mentioned.

I'd like to hear your opinion about docker builds, but i moved this to [an issue](https://github.com/neuronalmotion/qtrpi/issues/96) so maybe I can add this after the merge.

synapticvoid commented:
Alright, I've reviewed the change set. If you could do some minor tweaks, it would be perfect:
* Add the executable permission to `docker/build.sh`
* Update the `README.md` with the commands to build the Docker image
* Remove the `-j` flags on the `make` commands. It was my mistake, but it's actually useless, because omitting it will *not* limit the number of jobs

After that, I'll merge your PR and I'll take the time to clean up the repo to refer to the Docker build as the main one. Again, thanks for sharing your work :)

Commit:
Minor fixes

*Made build.sh executable
*remove -j flags from make

MoJo2600 commented:
I will update the README tomorrow
